hal/armv7m/timer.h

```diff
@@ -24,6 +24,12 @@
 #define TIMER_CYC2US(x) (x)
 
 
+extern void timer_jiffiesAdd(time_t t);
```

Comment:
I know that this function is not used on `imxrt` but this is a part of the interface.  I think it should be implemented as well.

Reply:
Done